onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -269,10 +269,33 @@ def aten_angle(self: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_any(self: TensorType) -> TensorType:
+@torch_op("aten::any", trace_only=True)
+def aten_any(self: BOOL, dim: Optional[int] = None, keepdim: bool = True) -> BOOL:
```

Comment:
Why do we specify the input is BOOL? Should not it be TTensor?

Reply:
any() means to check if any element in the tensor is not equal to 0, so finally it is a BOOL.